### ISSUE TITLE
SSE connection takes too long to open

### ISSUE THREAD

**Issue by NdozHH** [splitio/react-client#185]
Hi,

I'm trying to update the React SDK in my app but I noticed that the SSE connection takes more time to open in the latest version, look at this:

`"@splitsoftware/splitio-react": "1.2.6"`
With a fresh refresh the connection appears immediately 
![image](https://github.com/splitio/react-client/assets/25773289/1a29df3e-9c3a-49c0-9ed9-f826d1519e7d)

`"@splitsoftware/splitio-react": "1.11.0"` _Latest_
With the latest version an a fresh refresh the connection isn't opened immediately 
![image](https://github.com/splitio/react-client/assets/25773289/bc40f928-ec7e-48ad-8878-6b0275a2a52c)

But after 1 minute there it is
![image](https://github.com/splitio/react-client/assets/25773289/aeccffe6-3e83-4167-bdba-e1e2eda7558c)

Any ideas on why this is happening?
Can this behavior be changed to get the connection ready almost immediately?

**Comment by agustinona** [comment]
Hi @NdozHH since v1.3.0 the SDK connects to our auth service v2 (see: https://github.com/splitio/react-client/blob/master/CHANGES.txt), which applies a connection delay to ensure stability and protect our infrastructure.

It is not possible to change this through the SDK configuration, but it should not affect the SDK's functionality in any way. Are you experiencing any issues related to this?

**Comment by NdozHH** [comment]
Hi @agustinona , thank you for the explanation!

I was just curious about that, technically there isn't a problem but for example we can change something in a flag and if a user lands on the app in that moment, because of that time window of 1 minute he won't see the change reflected until the `sse` connection is opened an maybe a page reload.

**Comment by agustinona** [comment]
@NdozHH yes, that's correct. No page reload would be needed because when the sse connection is opened the SDK performs a full sync to ensure it gets any updates that may have happened during the connection delay window. It is worth noting that this is very unlikely to happen, and a 60 second delay is the absolute worst case scenario.
